bundle/manifests/customized-user-remediation.clusterserviceversion.yaml

```diff
@@ -362,6 +362,8 @@ spec:
   - baremetal
   - script
   - CUR
+  - customized-user-remediation
+  - Medik8s
```

Comment:
Usually, the keywords are lower-cased, but I am not sure whether this is an issue. 
https://sdk.operatorframework.io/docs/olm-integration/generation/#csv-fields